my_list.rb

```diff
@@ -0,0 +1,11 @@
+class MyList
```

Comment:
- [ ] Great job working on this milestone. You should use `require_relative` to import `MyEnumerable` since using `require_relative` for files in your directory is a ruby convention.  `require_relative` is used for referring to files within your directory.
`require_relative 'my_enumerable'`